models/src/main/java/com/vimeo/networking2/annotations/Internal.kt

```diff
@@ -0,0 +1,9 @@
+package com.vimeo.networking2.annotations
+
+/**
+ * Annotation to describe certain models as internal.
+ */
+@Target(AnnotationTarget.CLASS, AnnotationTarget.PROPERTY)
+@Retention(AnnotationRetention.SOURCE)
```

Comment:
The retention should be `RUNTIME` instead as `SOURCE` results in the annotation not being removed at compile time. You can see this in action if you decompile one of the annotated kotlin classes to java. When using `SOURCE`, the annotation is no longer visible in the class, while when using `RUNTIME` it becomes visible. The main purpose of this annotation is documentation and it therefore should be outputted to the class file.